src/testsrc/java/com/epam/jamp2/service/TestCalculationCommandExecutionService.java

```diff
@@ -0,0 +1,133 @@
+package com.epam.jamp2.service;
+
+import com.epam.jamp2.model.CalculationCommand;
+import com.epam.jamp2.model.Operation;
+import com.epam.jamp2.model.UnknownCurrencyException;
+import com.epam.jamp2.model.Value;
+import com.epam.jamp2.service.impl.CalculationCommandExecutionServiceImpl;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+import org.mockito.stubbing.OngoingStubbing;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.hamcrest.beans.HasPropertyWithValue.hasProperty;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.number.BigDecimalCloseTo.closeTo;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(value = Parameterized.class)
+public class TestCalculationCommandExecutionService {
```

Comment:
Minor. In my practice I mostly saw "Test" used as suffix in the end of test class name